python/tests/illarg_tests/str_type_fn.py

```diff
@@ -1,6 +1,26 @@
 from tests.aggregate_tests.aggtst_base import TstView
 
 
+# ASCII function
+class illarg_ascii_legal(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [{"str": 104}]
+        self.sql = """CREATE MATERIALIZED VIEW ascii_legal AS SELECT
+                      ASCII(str) AS str
+                      FROM illegal_tbl"""
+
+
+class illarg_ascii_wrong_type(TstView):
```

Comment:
the type is wrong but the test is still legal?
maybe we should not call it "wrong".
How about illarg_ascii_cast_legal?
"cast" because there is an implicit cast